api/migrations/schema/Version20240621195713.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240621195713 extends AbstractMigration {
+    public function getDescription(): string {
+        return '';
+    }
+
+    public function up(Schema $schema): void {
```

Comment:
maybe merge this with the migration [api/migrations/schema/Version20240620143000.php](https://github.com/ecamp/ecamp3/pull/5408/files#diff-c42fe2e1ba8e763c84b3558cf6fda6b2af2c1040215af731f291cf1502b77e77)?